scripts/build/environment.js

```diff
@@ -27,6 +27,8 @@
 
 'use strict';
 
+// TODO: remove this class. this is only used for webpack.
```

Comment:
I can add an issue for this, but I didn't wanna do this without chatting with the group about it. This seems to be unnecessary for webpack to work.

Reply:
yah, we can remove this in upcoming PR.

Reply:
created issue https://github.com/material-components/material-components-web/issues/4863